src/Bonsai.ML.LinearDynamicalSystems/Kinematics/KFModelParameters.cs

```diff
@@ -51,10 +51,10 @@ public class KFModelParameters
         private string fpsString;
 
         /// <summary>
-        /// x position at time 0
+        /// The initial x position.
```

Comment:
Depending on casing we would change this to `X` but see comment above, no need to change anything now.